drivers/i2c/i2c_ite_enhance.c

```diff
@@ -1286,6 +1303,33 @@ static int i2c_enhance_init(const struct device *dev)
 		return status;
 	}
 
+#ifdef CONFIG_I2C_TARGET_ALLOW_POWER_SAVING
+	if (config->target_enable) {
+		int ret;
```

Comment:
Is this variable necessary? There's `status` in scope, which is used for the same purpose